wremnants/plot_tools.py

```diff
@@ -7,37 +7,36 @@
 import numpy as np
 hep.style.use(hep.style.ROOT)
 
-def figureWithRatio(href, xlabel, ylabel, ylim, rlabel, rrange, xlim=None):
+def figureWithRatio(href, xlabel, ylabel, ylim, rlabel, rrange):
     hax = href.axes[0]
     width = math.ceil(hax.size/400)
+    print(width)
     fig = plt.figure(figsize=(8*width,8))
     ax1 = fig.add_subplot(4, 1, (1, 3)) 
     ax2 = fig.add_subplot(4, 1, 4) 
 
+    ax1.set_xlabel("")
     ax2.set_xlabel(xlabel)
-    ax1.set_xlabel(" ")
```

Comment:
I *think* this is needed for the xlabel to actually be removed for the not-ratio axis

Reply:
reverting the change